src/status_im/multiaccounts/core.cljs

```diff
@@ -201,3 +207,7 @@
   {:events [::update-local-picture]}
   [cofx pics]
   (multiaccounts.update/optimistic cofx :images pics))
+
+(comment
+  ;; Test seed for Dim Venerated Yaffle, it's not here by mistaka, this is just a test account
```

Comment:
why here ?

Reply:
This is inspired by a talk titled "[Running with Scissors](https://www.youtube.com/watch?v=Qx0-pViyIDU&ab_channel=StrangeLoop)" by Stuart Halloway. The idea is to capture your dev flow in git. 

While building the feature, I had to check the functioning of the `validate-mnemonic` function, so I captured it in a rich comment. If there is a bug, or we need to extend this feature, chances are that the ticket owner will need to check this function again. But the next time, it will be present in in a rich comment. 

But I also agree that it makes little sense in the this `ma.core` ns. So  I can get rid of it if there is any issue.